src/components/Seo.js

```diff
@@ -92,6 +92,17 @@ const SEO = ({ description, lang, meta, title }) => {
             src: url('/fonts/NotoKufiArabic-Bold.woff') format('woff'),
                   url('/fonts/NotoKufiArabic-Bold.ttf') format('truetype');
           }
+          @font-face {
```

Comment:
@sentayhuJo I added the fonts files themselves under `assets/fonts` folder, and use this [tool](https://onlinefontconverter.com/) to convert the `.ttf` file to `woff` format as well, which is used in modern browsers.

Reply:
Great !, +1 for the tools .